pyproject.toml

```diff
@@ -1,44 +1,82 @@
-[tool.poetry]
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
 name = "django-modeltranslation"
-version = "0.19.0"
+dynamic = ["version"]
 description = "Translates Django models using a registration approach."
-authors = ["Peter Eschler <peschler@gmail.com>"]
-maintainers = ["Serhii Tereshchenko <serg.partizan@gmail.com>"]
-packages = [
-    { include = "modeltranslation" },
+readme = "README.rst"
+authors = [
+    { name = "Peter Eschler", email = "peschler@gmail.com" },
+    { name = "Dirk Eschler", email = "eschler@gmail.com" },
+]
+maintainers = [
+    { name = "Sergiy Tereshchenko", email = "serg.partizan+modeltranslation@gmail.com" },
+]
+license = "BSD-3-Clause"
```

Comment:
@peschler @deschler Hey folks, I'm doing migration from poetry to uv here, and also updating build backend configuration.

And we had "New BSD" as license name in old config. But, this is not something exactly from this list https://spdx.org/licenses/

That's probably [BSD 3-Clause "New" or "Revised" License](https://spdx.org/licenses/BSD-3-Clause.html), aka `BSD-3-Clause`. Right?